packages/modules/manage-versions/src/components/ManageVersions/ManageVersions.tsx

```diff
@@ -140,11 +144,12 @@ export const ManageVersions = (props: ManageVersionsProps) => {
       versionClient
         .get(imodelId, { top: NAMED_VERSION_TOP, skip })
         .then((newVersions) => {
-          setVersionStatus(RequestStatus.Finished);
```

Comment:
Each data entry should have subRows property. If subRows has any items, then expander will be shown for that row.
Otherwise expander will not be visible in the table.
And if we consider status to be finished then it will show nv in table and after few seconds once getting changesets then expander icon is visible and that looks odd

Reply:
Well, we know that if we have NV, then FOR SURE it has changesets.
So have empty array, wont it show expander icon with empty array?

Reply:
No , not getting the expander icon and it's by design we must need to have data for subrows

Reply:
one thing we can do is instead of empty array we could set some dummy object in array and will show skeleton until we get changeset .

Reply:
I have define initialChangesets for the subRows and using state that determine subRows loaded or not to show loading for subrows.